questdb-rs/src/ingress/mod.rs

```diff
@@ -566,6 +593,8 @@ impl Buffer {
             state: BufferState::new(),
             marker: None,
             max_name_len: 127,
+            f64serializer: f64_binary_series,
+            version: LineProtocolVersion::V2,
```

Comment:
No, this needs to be an argument.
We _can't_ default to V2 just yet. It would break things too much.

Rather I think we can have a new utility:

```
let sender = Sender::from_conf(...);
let buffer = sender.new_buffer();  // version auto-populated.
```

I think it would be worth updating examples with this.

Reply:
Ensure that `Buffer::new()` documents and cross-references `sender.new_buffer()` and vice-versa.

Reply:
NB. The protocol should be added to `max_name_len` too.

In actual fact, `server.new_buffer()` should actually call `Buffer::with_max_name_len` under the covers, since this comes from config too.